src/main/java/net/minecraftforge/client/EnumSkyPass.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2018.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.client;
+
+public enum EnumSkyPass
```

Comment:
From what I've seen, Forge has mostly been moving away from enums prefixed with `Enum`. Could be changed to `SkyRenderPass`

Reply:
I didn't know that, thanks!